eip-template.md

```diff
@@ -9,6 +9,7 @@ type: <Standards Track, Meta, or Informational>
 category (*only required for Standards Track): <Core, Networking, Interface, or ERC>
 created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
 requires (*optional): <EIP number(s)>
+adoptable: true  # It is recommended that you keep this as true. See EIP-1#adoptable-header for more information.
```

Comment:
I am in favor of having an adoptable field, but I suggest keeping the template default as `false` until we see a trend that authors intentionally choose to flip it to `true`.  I worry if we create a field and default to true, the new comers of EIP authors will think it's a community etiquette to sign it off. Just like the "[ ] I agree " box in many terms of services.